Expensify/App
# [No QA] Update es translations for 'privacy' and 'invite'

cc @parasharrajat 
### Fixed Issues

$ https://github.com/Expensify/App/issues/6966

### Tests


1. Open app
2. Set language to Spanish
3. Navigate to Settings>Workspace>Connect bank account
4. Check that 'privacy' translates to `privacidad`
5. Navigate to Settings>Workspace> Members
6. Check that 'invite' translates to 'invitar'


### QA Steps

N / A

### Tested On


### Screenshots


#### Web

privacy => privacidad
![2022-01-14_14-27-21](https://user-images.githubusercontent.com/26260477/149595367-00864f53-6f77-4c00-a418-df4211cc56a4.png)
invite => invitar
![2022-01-14_14-28-33](https://user-images.githubusercontent.com/26260477/149595404-915e5a1d-4e28-4801-8441-20eac968ad82.png)

#### Mobile Web

privacy => privacidad
![2022-01-14_15-10-34](https://user-images.githubusercontent.com/26260477/149596848-af43bbd1-ca91-478c-94bc-7269c0603f77.png)
invite => invitar
![2022-01-14_15-11-03](https://user-images.githubusercontent.com/26260477/149596861-33fe7641-5cbb-4fe3-9afd-7ffda2586abb.png)

#### Desktop

privacy => privacidad
![2022-01-14_15-13-46](https://user-images.githubusercontent.com/26260477/149597505-20039b03-fdfd-43da-b86b-2fc52e00e16e.png)
invite => invitar
![2022-01-14_15-13-59](https://user-images.githubusercontent.com/26260477/149597514-ce3d8494-3a58-4d6e-9b66-4ebc9532d4f9.png)

#### iOS

privacy => privacidad
![2022-01-14_14-42-58](https://user-images.githubusercontent.com/26260477/149595572-993c87cb-8e09-4154-b916-6aae733dc559.png)
invite => invitar
![2022-01-14_14-43-25](https://user-images.githubusercontent.com/26260477/149595583-0ae2d832-55f1-40a7-9606-1ac5a853b5eb.png)


#### Android

privacy => privacidad
![2022-01-14_15-00-43](https://user-images.githubusercontent.com/26260477/149596143-0cd54616-bcf3-4fde-b24e-86681c5e2055.png)
invite => invitar
![2022-01-14_15-00-59](https://user-images.githubusercontent.com/26260477/149596156-ad2ab555-bc2f-4d7f-a4b0-1246a3f6a199.png)
